### PR TITLE
Cannot override default ghost button

### PR DESCRIPTION
**Pull request by iharpeshka**
I needed to override color default ghost button, but I did not want to do this with "css overide"....

### CODE REVIEW

File: components/button/style/index.less [hunk 1]
```diff
@@ -142,9 +142,7 @@
   }
 
   &-background-ghost {
-    background: transparent!important;
```
Comment (benjycui) [inline components/button/style/index.less]: 我不懂，你为什么要覆盖 Ghost button 的背景色？重设计定义 Ghost 必须没有背景色。 https://ant.design/components/button-cn/#components-button-demo-ghost

Comment (iharpeshka) [inline components/button/style/index.less]: I want to replace the button color !!

Comment (iharpeshka) [inline components/button/style/index.less]: In the current implementation, it is always white color text and color btn? But why is it so? Ghost warning btn i can override....

**Comment by iharpeshka** [comment]
I do a default (white) button on a white background... and i already define primary ghost button with blue color

File: components/button/style/index.less (hunk 1 above)
Comment (iharpeshka) [inline components/button/style/index.less]: I do a default (white) button on a white background... and i already define primary ghost button with blue color

Comment (xueqingxiao) [inline components/button/style/index.less]: @iharpeshka Cause we must felllow design spec, if you want chang backgroud color & text color, you can setup styles by yourself.

**Pull request closed without merging**